executions/graphql-kotlin-transaction-loader-instrumentation/README.md

```diff
@@ -65,7 +65,7 @@ val queries = [
 ]
 
 val graphQLContext = mapOf(
-    TransactionBatcher::class to transactionBatcher,
+    DataLoaderRegistry::class to dataLoaderRegistry,
```

Comment:
should we populate `KotlinDataLoaderRegistry`? i.e. if it is regular data loader registry you won't get all the dispatch info?

Reply:
Good catch!, thanks!